### DISCUSSION TITLE
モバイル学習コンテンツで、認証例として、以下のリファレンスの参照があっても良い

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#261] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/todo-app/app-spec
ToDoアプリの仕様

> アプリを利用するためには、アカウントを登録してユーザを認証する必要があります。 ただし、現在のバージョンではアプリのメモリ上にしか情報をもたないため、認証はダミーとします。

上記の記載があるので、認証例として、以下のリファレンスの参照があっても良い。

https://ws-4020.github.io/mobile-app-crib-notes/reference/auth/overview
認証

**Comment by nagano-ryo** [comment]
- ハンズオンからリファレンスへのリンクを張ること自体はOK。
- OKだが、リンクだけ張るのはNG。（リファレンスは方式を決める人が見るもの。ハンズオンとレベルが異なる。）

**Comment by nagano-ryo** [comment]
pull request
https://github.com/ws-4020/mobile-app-crib-notes/pull/276

**Comment by tani-takanori** [comment]
リファレンスを貼りたいという動機がmobile-app-crib-notesにあるコンテンツへの導線を増やしたい（良いコンテンツがもっとあることを知ってほしい）ということであれば、ここにリンクを貼るよりサイトマップとかがあったほうがいいと思う。

もしくは応用編や参考情報に貼る？

学習中に必要じゃないけどInputしてほしいものを増やすと、わからないものが増えてしまうので一つづつ学習して欲しい。

**Reply by creature-water-valley** [reply]
> 学習中に必要じゃないけどInputしてほしいものを増やすと、わからないものが増えてしまうので一つづつ学習して欲しい。

これに同意なので、リファレンスの参照はなくても良い気がしました。
